pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -113,7 +113,7 @@ func TestNewCredentialFromLinkedDataProof_Ed25519Signature2018_Transmute(t *test
 
 	vcWithLdp, _, err := NewCredential([]byte(vcFromTransmute),
 		WithEmbeddedSignatureSuites(sigSuite),
-		WithPublicKeyFetcher(SingleKey(pubKeyBytes, kms.ED25519)))
+		WithPublicKeyFetcher(SingleKey(pubKeyBytes, "Ed25519Signature2018")))
```

Comment:
I would make "Ed25519Signature2018" a const in the test file.

Reply:
Ah, I just found that @fqutishat already added the names of the signature suites as constants. I will use them.